testsuite/features/support/commonlib.rb

```diff
@@ -219,6 +219,21 @@ def get_client_type(name)
   end
 end
 
+def client_public_ip(host)
+  node = get_target(host)
+  interfaz = 'eth0'
+  case host
+  when /^sle/, /^ceos/
+    interfaz = 'eth0'
+  when /^ubuntu/
+    interfaz = 'ens3'
+  end
+  output, code = node.run("ip address show dev #{interfaz} | grep 'inet '")
```

Comment:
`hostname -i` can be good enough? Or it might be the case that the first IP is not the IP we want? :thinking: